src/test/java/com/softserveinc/dokazovi/service/impl/PostServiceImplTest.java

```diff
@@ -1723,4 +1797,30 @@ void setPublishedAtTest() {
 		Mockito.when(postRepository.findById(1)).thenReturn(Optional.of(postEntity));
 		assertTrue(postService.setPublishedAt(1, postPublishedAtDTO));
 	}
+
+	@Test
+	void setPublishedAt_WhenPost_NotFound_ThrowException() {
+		Integer postId = 0;
+		Timestamp publishedAt = Timestamp.valueOf(
+				LocalDateTime.of(
+						LocalDate.of(
+								2002, Month.JANUARY, 14),
```

Comment:
Please move it on the same line with `LocalDate.of(`

Reply:
Fixed.